plugin.go

```diff
@@ -83,13 +83,23 @@ func (p Plugin) Exec() error {
 	return client.PostMessage(&payload)
 }
 
+func getRef(build Build) string {
+	ref := ""
+	if build.Commit != "" {
+		ref = build.Commit[:8]
+	} else {
+		ref = build.Tag
+	}
+	return ref
+}
```

Comment:
also maybe we want a special message for deployments? I'm not sure the message makes sense with tag, followed by branch